README.md

```diff
@@ -1 +1,37 @@
-# ctd-omega-editorial-frontend
\ No newline at end of file
+# ctd-omega-editorial-frontend
+
+This project is an editorial frontend for managing The National Archives Pan-Archival Catalogue (PAC).
+
+The project is developed in Scala Play framework with govuk-frontend toolkit using  Scala Twirl implementation of govuk-frontend and hmrc-frontend for building the frontend web application.
```

Comment:
The official name for the toolkit is GOV.UK Design System and you want to link that to here https://design-system.service.gov.uk/. Also I think it would be better to say that we are using the Twirl implementation of the design system provided by HMRC at https://github.com/hmrc/play-frontend-hmrc as we are not actually using the hmrc-frontend components in there.